go/test/endtoend/vtgate/aggr_test.go

```diff
@@ -40,3 +40,45 @@ func TestAggregateTypes(t *testing.T) {
 	assertMatches(t, conn, "select val1, count(distinct val2) k, count(*) from aggr_test group by val1 order by k desc, val1 limit 4", `[[VARCHAR("c") INT64(2) INT64(2)] [VARCHAR("a") INT64(1) INT64(2)] [VARCHAR("b") INT64(1) INT64(1)] [VARCHAR("e") INT64(1) INT64(2)]]`)
 	exec(t, conn, "delete from aggr_test")
 }
+
```

Comment:
nit: doesn't feel like the right file for these tests

Reply:
moved